.github/workflows/maven.yml

```diff
@@ -94,7 +170,7 @@ jobs:
         if: contains(matrix.os, 'win') == false
         run: chmod +x ./mvnw
       - name: Build with Maven
-        run: ./mvnw verify --file pom.xml -Pcoverage
+        run: ./mvnw verify --file pom.xml -Pcoverage,kotlin-1.9
```

Comment:
`kotlin-1.9` needs to be explicitly activated here, as [documented here](https://maven.apache.org/guides/introduction/introduction-to-profiles.html#Explicit_profile_activation):

> This profile will automatically be active for all builds unless another profile in the same POM is activated using one of the previously described methods. All profiles that are active by default are automatically deactivated when a profile in the POM is activated on the command line or through its activation config.